azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServerModule.java

```diff
@@ -87,6 +95,10 @@ protected void configure() {
     bind(FlowTriggerInstanceLoader.class).to(JdbcFlowTriggerInstanceLoaderImpl.class);
     bind(ExecutorManagerAdapter.class).to(resolveExecutorManagerAdaptorClassType());
     bind(WebMetrics.class).to(resolveWebMetricsClass()).in(Scopes.SINGLETON);
+    bind(ImageVersionService.class).to(ImageVersionServiceImpl.class);
+    bind(ImageVersionDao.class).to(ImageVersionDaoImpl.class);
+    bind(ImageTypeService.class).to(ImageTypeServiceImpl.class);
```

Comment:
Can you please move type before version? Just good for readability to keep the order maintained considering dependency.

Reply:
Done.